driver.sh

```diff
@@ -129,6 +129,78 @@ build_linux() {
   # is good about figuring out what needs to be rebuilt
   [[ -n "${cleanup:-}" ]] && mako_reactor mrproper
   mako_reactor ${config}
+  # From https://github.com/groeck/linux-build-test/blob/master/rootfs/scripts/common.sh
+  echo "CONFIG_EXPERT=y
+CONFIG_DEBUG_KERNEL=y
+CONFIG_LOCK_DEBUGGING_SUPPORT=y
+CONFIG_DEBUG_RT_MUTEXES=y
+CONFIG_DEBUG_SPINLOCK=y
+CONFIG_DEBUG_MUTEXES=y
+CONFIG_DEBUG_WW_MUTEX_SLOWPATH=y
+CONFIG_DEBUG_LOCK_ALLOC=y
+CONFIG_DEBUG_LOCKDEP=y
+CONFIG_DEBUG_ATOMIC_SLEEP=y
+CONFIG_CRYPTO_MANAGER_DISABLE_TESTS=y
+CONFIG_CRC32_SELFTEST=y
+CONFIG_DEBUG_LOCKING_API_SELFTESTS=y
+CONFIG_DEBUG_NMI_SELFTEST=y
+CONFIG_DEBUG_RODATA_TEST=y
+CONFIG_DEBUG_TLBFLUSH=y
+CONFIG_DMATEST=y
+CONFIG_GLOB_SELFTEST=y
+CONFIG_PCI_EPF_TEST=y
+CONFIG_PCI_ENDPOINT_TEST=y
+CONFIG_PROVE_LOCKING=y
+CONFIG_PROVE_RCU=y
+CONFIG_RBTREE_TEST=y
+CONFIG_RCU_EQS_DEBUG=y
+CONFIG_STATIC_KEYS_SELFTEST=y
+CONFIG_STRING_SELFTEST=y
+CONFIG_TEST_BITMAP=y
+CONFIG_TEST_FIRMWARE=y
+CONFIG_TEST_SORT=y
+CONFIG_TEST_SYSCTL=y
+CONFIG_TEST_UUID=y
+CONFIG_TORTURE_TEST=y
+CONFIG_USB_TEST=y
+CONFIG_USB_EHSET_TEST_FIXTURE=y
+CONFIG_USB_LINK_LAYER_TEST=y
+CONFIG_WW_MUTEX_SELFTEST=y
+CONFIG_BLK_DEV_INITRD=y
+CONFIG_DEVTMPFS=y
+CONFIG_DEVTMPFS_MOUNT=y
+CONFIG_SCSI=y
+CONFIG_BLK_DEV_SD=y
+CONFIG_SCSI_LOWLEVEL=y
+CONFIG_SCSI_DC395x=y
+CONFIG_SCSI_AM53C974=y
+CONFIG_SCSI_SYM53C8XX_2=y
+CONFIG_MEGARAID_SAS=y
+CONFIG_FUSION=y
+CONFIG_FUSION_SAS=y
+CONFIG_MMC=y
+CONFIG_MMC_SDHCI=y
+CONFIG_MMC_SDHCI_PCI=y
+CONFIG_BLK_DEV_NVME=y
+CONFIG_USB=y
+CONFIG_USB_XHCI_HCD=y
+CONFIG_USB_EHCI_HCD=y
+CONFIG_USB_OHCI_HCD=y
+CONFIG_USB_STORAGE=y
+CONFIG_USB_UAS=y
+CONFIG_VIRTIO=y
+CONFIG_VIRTIO_PCI=y
+CONFIG_VIRTIO_PCI_LEGACY=y
+CONFIG_VIRTIO_BALLOON=y
+CONFIG_VIRTIO_MMIO=y
+CONFIG_BLK_MQ_VIRTIO=y
+CONFIG_VIRTIO_BLK=y
+CONFIG_VIRTIO_BLK_SCSI=y
+CONFIG_SCSI_VIRTIO=y" >> .config
```

Comment:
Put this is a different file checked in (`git add ...`), then copy it over to `.config`.

Reply:
as in, this should be a separate config file, rather than a big string literal in a shell script. Then the shell script can simply do `cp groeck_config .config` for a given target.

Reply:
These configs are meant to supplement a defconfig, not be a complete config fragment but I agree that they should be in a separate file then concatenated onto `.config`.

Reply:
Yes please.